hikari/channels.py

```diff
@@ -1288,7 +1288,7 @@ class GuildNewsChannel(TextableGuildChannel):
 
 
 @attr.define(hash=True, kw_only=True, weakref_slot=False)
-class GuildVoiceChannel(GuildChannel):
+class GuildVoiceChannel(TextableGuildChannel):
```

Comment:
You need to add the `last_message_id` field to this class

Reply:
this still needs to be added

Reply:
Whoops, looks like I completely forgot about this PR, should be resolved now.